src/main/java/usecases/suggest_title/StInteractor.java

```diff
@@ -0,0 +1,79 @@
+package usecases.suggest_title;
+import entities.SuggestedTitleChecker;
+import usecases.Response;
+
+import java.util.Arrays;
+
+/**
+ * The interactor for this use case. Contains a thread that carries out the processes involved in suggesting a story\
+ * title.
+ */
+public class StInteractor {
+    StOutputBoundary pres;
+    StGateway repo;
+    SuggestedTitleChecker titleChecker;
+
```

Comment:
Add a constructor to take in and set the above attributes. Also please add ThreadRegister attribute as shown in blue in the clean arch doc and take it in and set it in the constructor.